client/my-sites/sidebar/test/sidebar.jsx

```diff
@@ -25,9 +25,10 @@ describe( 'MySitesSidebar', () => {
 		MySitesSidebar = require( '../sidebar' ).MySitesSidebar;
 	} );
 
-	describe( '#isItemLinkSelected()', () => {
+	describe.only( '#isItemLinkSelected()', () => {
```

Comment:
I've just noticed this - usually pretty vigilant with this :O  
Will be fixed up soon!